### PR TITLE
Arm templates modifications

### PR DESCRIPTION
**Pull request by irinatarn**

### CODE REVIEW

File: arm/deploy_ad_app.ps1 [hunk 1]
```diff
@@ -185,14 +185,14 @@ $tenantShortName = GetTenantShortName -TenantId $tenantId
 # Outputs
 Write-Host "== Collect shared parameters"
 Write-Host "Azure AD application name:"$prefix
-Write-Host "Azure AD application client id:"$appAd.ApplicationId
+Write-Host "Azure AD application client id:"$appAd.AppId
```
Comment (satyamamit) [inline arm/deploy_ad_app.ps1:188]: any idea why deployment for merchantx and Prod didn't gave errors ?

File: arm/functions.ps1 [hunk 2]
```diff
@@ -202,6 +202,12 @@ function CreateMapApp {
     )
 
     Write-Host "= Create application"
+	Write-Host $mapAppName
+	
+	Write-Host $mapAppSecurePassword
```
Comment (satyamamit) [inline arm/functions.ps1:207]: writting password on host ?

File: arm/functions.ps1 [hunk 3]
```diff
@@ -202,6 +202,12 @@ function CreateMapApp {
     )
 
     Write-Host "= Create application"
+	Write-Host $mapAppName
```
Comment (satyamamit) [inline arm/functions.ps1:205]: I think all these statements were for testing stuff.

File: arm/deploy_frontend.ps1 [hunk 4]
```diff
@@ -72,15 +72,9 @@ $json.clientId = $clientId
 $json.tenant = $tenantId
 $json.mapClientId = $mapClientId
 
-$json | ConvertTo-Json | Set-Content 'config.json'
```
Comment (satyamamit) [inline arm/deploy_frontend.ps1:75]: was previous statement not working ? I think the problem was AppId was not getting proper value, once you changed to AppId code, I think it should have been working now.

File: arm/templates/linuxApp.json [hunk 5]
```diff
@@ -215,6 +215,10 @@
                 {
                   "name": "KEYVAULT_ENDPOINT",
                   "value": "[parameters('keyVaultEndpoint')]"
+                },
+				{
+                  "name": "WEBSITE_RUN_FROM_PACKAGE_HEALTH_CHECK",
+                  "value": "1"
```
Comment (satyamamit) [inline arm/templates/linuxApp.json:221]: was it 1 or 0.

**Review by satyamamit — DISMISSED** [review]

File: arm/deploy_ad_app.ps1 (hunk 1 above)
Comment (irinatarn) [reply arm/deploy_ad_app.ps1:188]: I was using old azure cli thingy - once I upgraded it started failing for me.

File: arm/deploy_frontend.ps1 (hunk 4 above)
Comment (irinatarn) [reply arm/deploy_frontend.ps1:75]: Nope.
I tested that multiple times with Chad:)
This particular line was saving config.json as empty hence our propblems.

File: arm/templates/linuxApp.json (hunk 5 above)
Comment (irinatarn) [reply arm/templates/linuxApp.json:221]: 1

**Review by chadsowald — APPROVED** [review]
Looks good to me.  Nice find on the AppId!

**Pull request merged**